MediaPlayer/MediaPlayer.module.less

```diff
@@ -0,0 +1,25 @@
+@import "../styles/skin.less";
+
+.mediaPlayer {
+	.times {
+		display: flex;
+		justify-content: space-between;
+		position: relative;
+		bottom: 15px;
+	}
+
+	.applySkins({
+		height: 132px;
+
+		.times {
```

Comment:
I think this class should move to `MediaPlayer/Times.module.less`.

Reply:
If I move this back to `MediaPlayer/Times.module.less`, I would need to wrap again `Times.js` with `Skinnable`. I referenced `MediaPlayer.module.less` in `Times.js`, so that's why I defined it here. What do you think?

Reply:
Oh really? then I prefer to have `MediaPlayer/Times.modules.less`. Sorry!

Reply:
I checked again and it worked. I'm pushing now the changes. Sorry!